packages/keyring-snap-bridge/package.json

```diff
@@ -51,9 +52,6 @@
     "typedoc": "^0.25.13",
     "typescript": "~5.4.5"
   },
-  "peerDependencies": {
```

Comment:
Why this change? I think we still use some `keyring-api` types in our public API 🤔

Reply:
That was mainly to make the preview builds work (so I can use them in the extension afterward)

But yes, I'll definitely revert those changes before merging 👍

Reply:
I did revert that back now!